aldryn_events/south_migrations/0016_create_default_namespaces.py

```diff
@@ -11,6 +12,11 @@ def forwards(self, orm):
         # Note: Don't use "from appname.models import ModelName".
         # Use orm.ModelName to refer to models in this application,
         # and orm['appname.ModelName'] for models in other applications.
+
+        # in order to fix this migration for sqlite3 we need to enable
+        # transaction autocommit, otherwise it is broken
+        if connection.vendor == 'sqlite':
```

Comment:
OK, but if we find that we need to do this again and again and again maybe create a specialised DataMigration class that does this?